src/components/tag/index.tsx

```diff
@@ -1,61 +1,82 @@
-import qtMerge from 'qtMerge'
-import { type QuillSvgProps } from '@deriv/quill-icons'
+import React, { forwardRef, HTMLAttributes } from 'react'
+import { QuillSvgProps } from '@deriv/quill-icons'
 import {
-  TagSizes,
-  TagIconSizes,
-  TagIcons,
+  TagClassNames,
+  TagCustomIconColors,
   TagIconColors,
-  TagColors,
-  tagBaseClassnames,
+  TagIcons,
+  TagIconSizes,
 } from './tag.classnames'
-import { HTMLAttributes, forwardRef } from 'react'
 
-export type TagType = 'error' | 'warning' | 'success' | 'info'
-export type TagSize = 'xs' | 'sm' | 'md' | 'lg'
-export type TagVariant = 'fill' | 'outline'
+export type TPreset = 'custom' | 'success' | 'error' | 'warning' | 'info'
```

Comment:
Can we move these to a separate types file?

Reply:
I think we should wait for @mohsen-deriv  to see what he has plan for this.